components/x-podcast-launchers/src/PodcastLaunchers.scss

```diff
@@ -55,10 +55,18 @@ $o-buttons-is-silent: true;
 	margin: 0 0 24px;
 }
 
+
+/*
+ Hide podcast links at a wide screen width. We only do this when the running application
+ isn't the App so that on any native device you'll always be able to launch a podcast app
+*/
+.hidePodcastLinkAtWide {
+		@include oGridRespondTo(L) {
```

Comment:
this will also hide the buttons on landscape tablets, is that intentional?

Reply:
😳 No what I've done here is renamed the class in the CSS but not renamed it in the jsx

```
<li key={name} className={isApp ? "" : styles.hidePodcastLinkAtWide}>
```

I think this solution is a bit gross btw - so if you'd have done it a different more elegant way please tell me!

Reply:
we don't really use anything other than viewport for detecting mobile anywhere else (the other comparable case i can think of is hiding the Whatsapp share icon which [uses the `S` breakpoint](https://github.com/Financial-Times/next-article/blob/cf51a94382cb9fb1b98dfb144781e456565736ca/client/components/share/_mixins.scss#L19-L22))